awslnx-systemd/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM amazonlinux:2.0.20190508
```

Comment:
I am not sure, but this file isn't it the file generated from your Jinja template ? if yes, maybe isn't it necessary to commit, true ?